src/SecTester.Scan/Har/UploadHarOptions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Text.Json.Serialization;
+
+namespace SecTester.Scan.Har;
+
+public class UploadHarOptions
+{
+  public HarContent HarContent { get; set; }

```

Comment:
Use init only setter in immutable types

```suggestion
  public HarContent HarContent { get; init; }

```